mmdeploy/codebase/mmcls/__init__.py

```diff
@@ -1,3 +1,3 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from . import models  # noqa: F401,F403
 from .deploy import *  # noqa: F401,F403
```

Comment:
should update `mmdeploy/codebase/mmaction/models/recognizers/__init__.py` as well